packages/gcc8.rb

```diff
@@ -0,0 +1,160 @@
+require 'package'
+
+class Gcc8 < Package
+  description 'The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Ada, and Go.'
+  homepage 'https://www.gnu.org/software/gcc/'
+  version '8.2.0'
+  source_url 'https://ftpmirror.gnu.org/gcc/gcc-8.2.0/gcc-8.2.0.tar.xz'
+  source_sha256 '196c3c04ba2613f893283977e6011b2345d1cd1af9abeac58e916b1aab3e0080'
+
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/gcc8-8.2.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/gcc8-8.2.0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/gcc8-8.2.0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/gcc8-8.2.0-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '536323fc5dd35e53665a19e65c636f288153efe4cdf31617c32103c64593b285',
+     armv7l: '536323fc5dd35e53665a19e65c636f288153efe4cdf31617c32103c64593b285',
+       i686: 'e37ca8d49198beaa9ba582fc25fa598ab3cd5ed7e46f96a03176fb9002cc6f7b',
+     x86_64: '6d982f3c35acd3a738e176a3f3b8c4ea093c7be86882153a7711841cf0c80737',
+  })
+
+  depends_on 'unzip' => :build
+  depends_on 'gawk' => :build
+  depends_on 'dejagnu' => :build # for test
+  depends_on 'gcc7' => :build # gcc version 7.4.0
+  depends_on 'icu4c' => :build # icu version 62.1
+  depends_on 'python27' => :build
+  depends_on 'python3' => :build
+
+  depends_on 'binutils'
+  depends_on 'gmp'
+  depends_on 'mpfr'
+  depends_on 'mpc'
+  depends_on 'isl'
+  depends_on 'cloog'
+  depends_on 'glibc'
+
+  def self.patch
+    # See https://gcc.gnu.org/bugzilla/show_bug.cgi?id=86724#c4.
+    system "sed -i '40i#include <isl/id.h>' gcc/graphite.h"
+    system "sed -i '41i#include <isl/space.h>' gcc/graphite.h"
```

Comment:
You can always use my patch... Also, very bad idea to hardcode the line names.

Reply:
Where was your patch when I was trying to compile on i686?  This fixes the issue and using line numbers works.  This "patch" may not even be needed for the next version.

Reply:
Probably not. They already have it fixed in `master`.